Tools/Scripts/run-jsc-stress-tests

```diff
@@ -2813,6 +2813,16 @@ def copyBundleToRemote(remoteHost)
     mysys(["scp"] + SSH_OPTIONS_DEFAULT + (remoteHost.identity_file_path ? ["-i", remoteHost.identity_file_path] : []) + ["-P", remoteHost.port.to_s, ($outputDir.dirname + $tarFileName).to_s, "#{remoteHost.user}@#{remoteHost.host}:#{remoteHost.remoteDirectory}"])
 end
 
+def getLocalRemoteResultPath(remoteHost)
+    $outputDir.dirname + "#{remoteHost.host}-#{remoteHost.port}"
+end
+
+def copyRemoteResultToHost(remoteHost)
+    hostResultFolder = getLocalRemoteResultPath(remoteHost)
+    Dir.mkdir hostResultFolder unless hostResultFolder.directory?
+    mysys(["scp"] + SSH_OPTIONS_DEFAULT + (remoteHost.identity_file_path ? ["-i", remoteHost.identity_file_path] : []) + ["-P", remoteHost.port.to_s, "#{remoteHost.user}@#{remoteHost.host}:#{remoteHost.remoteDirectory}/#{$outputDir.basename}/.runner/#{STATUS_FILE}", "#{getLocalRemoteResultPath(remoteHost).to_s}/#{STATUS_FILE}" ])
+end
```

Comment:
There's nothing wrong with this per-se, but I wonder if it would be more straightforward to use `sshRead` along with `cat` in `getStatusMap` instead of invoking `scp` directly here and using a temporary file.

Reply:
Using driver host to read will be more fast, since the remote device is often an embedded device with limited CPU and IO speed.

Reply:
Just to be clear, what I was suggesting was using the equivalent of `ssh host cat file` instead of `scp host:file .`, simply because we already have a mechanism for the former. So I don't think the embedded device would need to use more CPU time. Not that important though.